### DISCUSSION TITLE
Does python-mysql-replication support db encodings other than utf8/utf8mb4?

### DISCUSSION THREAD

**Discussion by mengfp** [julien-duponchelle/python-mysql-replication#613] [Q&A]
Does it work if there're non-utf8 characters in the binlog, like Chinese characters (GBK) ?

**Comment by sean-k1** [comment]
@mengfp sorry i can not test this one.
but I think we do not support GBK.

we set connection like this 
```
self.__connection_settings.setdefault("charset", "utf8")
```
